metis/bin/metis_client

```diff
@@ -354,8 +350,12 @@ class MetisClient < Client
     raise MetisClient::Error, "Could not upload file #{upload.file_path}\n#{error(response)}" unless response.code == "200"
   end
 
+  def encode_path(path)
+    path.split('/').map { |v| URI.encode_www_form_component(v) }.join('/')
```

Comment:
err wups, wrote this same method the other day... Probably should prefer URI.encode_www_form_component to CGI.escape.